lib/samson/logging.rb

```diff
@@ -3,7 +3,7 @@
 
 if Samson::EnvCheck.set?("RAILS_LOG_TO_STDOUT")
   # heroku and docker: dump everything to stdout
-  config.logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT))
+  config.logger = ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new(STDOUT))
```

Comment:
sorry, I wrote it in my commit message on https://github.com/zendesk/samson/pull/1264/commits/dab7c9fd5d1cc0efda5b7450a46e582f6a2ec72c